From 6853ad6f9efe82cd1aa55c9ec260f77f89098aa1 Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Sun, 19 Feb 2017 19:49:54 +0200
Subject: [PATCH 1375/2241] arm: defconfig: enable dm-crypt

dm-crypt is a disk encryption system using the kernels crypto API framework
and device mapper subsystem.
With dm-crypt, administrators can encrypt entire disks, logical volumes,
partitions, but also single files.

Change-Id: Iceb25d73e9240230b21085e9f3d0ba417f7b8f63
Signed-off-by: Igal Liberman <igall@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/36884
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Neta Zur Hershkovits <neta@marvell.com>
---
 arch/arm/configs/mvebu_v7_lsp_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/configs/mvebu_v7_lsp_defconfig b/arch/arm/configs/mvebu_v7_lsp_defconfig
index 31d1e3c..4722e80 100644
--- a/arch/arm/configs/mvebu_v7_lsp_defconfig
+++ b/arch/arm/configs/mvebu_v7_lsp_defconfig
@@ -76,6 +76,8 @@ CONFIG_BLK_DEV_MD=y
 CONFIG_MD_RAID0=y
 CONFIG_MD_RAID1=y
 CONFIG_MD_RAID456=y
+CONFIG_BLK_DEV_DM=y
+CONFIG_DM_CRYPT=y
 CONFIG_NETDEVICES=y
 CONFIG_BONDING=y
 CONFIG_NET_DSA_MV88E6XXX=y
-- 
2.7.4

